station-renderer-arsenic/src/main/java/net/modificationstation/stationapi/impl/client/arsenic/renderer/aocalc/LightingCalculatorImpl.java

```diff
@@ -80,6 +80,13 @@ private int toIndex(int x, int y, int z) {
     }
 
     public void calculateForQuad(BakedQuad q) {
+        float emission = q.getEmission();
+        
+        if (emission >= 1) {
+            Arrays.fill(light, emission);
```

Comment:
What's the point of filling the array if the values are going to be overwritten anyway?